### DISCUSSION TITLE
Rate limiting

### DISCUSSION THREAD

**Discussion by pnewell** [DIYgod/RSSHub#11461] [Q&A] [answered]
### What feature is it?

I am trying to implement a new route and running into issues with rate limits. The Promise.all for loading all the articles is leading to some 400. Is there a recommend official way to handle this?

### What problem does this feature solve?

Rate limits when loading article content

### This is not a duplicated feature request or new RSS proposal

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this feature has not already been requested and this is not a [new RSS proposal](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_en.yml).

**Comment by TonyRL** [comment, marked as answer]
FYI: https://github.com/DIYgod/RSSHub/search?q=asyncpool
